src/lightgbm/src/main/scala/RegressorUtils.scala

```diff
@@ -0,0 +1,8 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
```

Comment:
This is great but does it break backwards compatibility? Can I load models saved using older versions of MMLSpark?

Reply:
this only affects LightGBM so no backwards compat issues here